### PR TITLE
Fail gracefully if trying to read password in msys

### LINKED ISSUE #437: vegawallet running in Git Bash does not accept user input
vegawallet running on Git Bash outputs an error when requesting input if run from git bash on Windows. 

Repro:
1. `go install code.vegaprotocol.io/vegawallet@v0.11.0`
2. `vegawallet create --wallet TEST`

Expected behaviour:
Wallet created. All output is readable, no visual artifacts or spurious characters.

Observed behaviour:
"The handle is invalid" error is displayed to user, and no wallet is created.
image

Notes:
- Running vegawallet in Command Prompt or Powershell works fine, other than https://github.com/vegaprotocol/vegawallet/issues/436
- Confirmed that you can workaround this issue by using --passphrase-file for this situation.

### PR DESCRIPTION
**Pull request by pscott31**
Closes #437

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,8 @@
 - [420](https://github.com/vegaprotocol/vegawallet/pull/420) - Verify initialisation state on service run
 - [421](https://github.com/vegaprotocol/vegawallet/pull/421) - Improve help texts
 - [423](https://github.com/vegaprotocol/vegawallet/pull/423) - Support launching the token dApp in `service run` command
+- [427](https://github.com/vegaprotocol/vegawallet/pull/427) - Sign MacOS binaries
+- [444](https://github.com/vegaprotocol/vegawallet/pull/444) - Fail gracefully when trying to get password input on msys
```
Comment (ValentinTrinque) [inline CHANGELOG.md:49]: This is at the wrong place, it should be under `Unreleased`

Comment (pscott31) [reply CHANGELOG.md]: My bad, will fix

File: cmd/flags/passphrase.go [hunk 2]
```diff
@@ -102,7 +103,15 @@ func ReadPassphraseInputWithOpts(withConfirmation bool) (string, error) {
 	return passphrase, nil
 }
 
+func runningInMsys() bool {
+	ms := os.Getenv("MSYSTEM")
+	return ms != ""
```
Comment (ValentinTrinque) [inline cmd/flags/passphrase.go:108]: Beware, you are not checking for errors here. Also, ou may want to consider `os.LookupEnv()` instead. 

https://pkg.go.dev/os#LookupEnv

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]

File: cmd/flags/passphrase.go (hunk 2 above)
Comment (ValentinTrinque) [reply cmd/flags/passphrase.go:108]: Sorry, my mistake, it's a false statement. Ignore this.

**Review by ValentinTrinque — APPROVED** [review]

**Pull request merged**